### PR TITLE
UI Automation Updates.

### PR DESCRIPTION
**Pull request by trwalke**
Removing dependency on IsAppcenterBuild.
Updating mac os image

### CODE REVIEW

File: src/client/Microsoft.Identity.Client/Platforms/iOS/SystemWebview/SystemWebUI.cs [hunk 1]
```diff
@@ -97,25 +94,6 @@ public void Authenticate(Uri authorizationUri, Uri redirectUri, RequestContext r
 
                     sfAuthenticationSession.Start();
                 }
-#else
-                if (UIDevice.CurrentDevice.CheckSystemVersion(11, 0))
-                {
```
Comment (jennyf19) [inline src/client/Microsoft.Identity.Client/Platforms/iOS/SystemWebview/SystemWebUI.cs:102]: don't we need this still? for iOS 11?

File: src/client/Microsoft.Identity.Client/Platforms/iOS/EmbeddedWebview/MsalAuthenticationAgentUIViewController.cs [hunk 2]
```diff
@@ -53,12 +53,12 @@ public override void ViewDidLoad()
         protected WKWebView PrepareWKWebView()
         {
             WKWebViewConfiguration wkconfg = new WKWebViewConfiguration() { };
-#if !IS_APPCENTER_BUILD
```
Comment (bgavrilMS) [inline src/client/Microsoft.Identity.Client/Platforms/iOS/EmbeddedWebview/MsalAuthenticationAgentUIViewController.cs:56]: yay!

File: src/client/Microsoft.Identity.Client/Platforms/iOS/SystemWebview/SystemWebUI.cs (hunk 1 above)
Comment (trwalke) [reply src/client/Microsoft.Identity.Client/Platforms/iOS/SystemWebview/SystemWebUI.cs:102]: you are correct. adding back in

File: src/client/Microsoft.Identity.Client/Platforms/iOS/EmbeddedWebview/MsalAuthenticationAgentUIViewController.cs (hunk 2 above)
Comment (bgavrilMS) [inline src/client/Microsoft.Identity.Client/Platforms/iOS/EmbeddedWebview/MsalAuthenticationAgentUIViewController.cs:56]: I think this flag also appears in the some of the YAML build defintions..

Comment (trwalke) [reply src/client/Microsoft.Identity.Client/Platforms/iOS/EmbeddedWebview/MsalAuthenticationAgentUIViewController.cs:56]: yes, it is still needed for the broker test app that we create for the broker team to test for us. However, these are no longer needed in any of the shipped product code.

**Pull request merged**